HOWTO.md

```diff
@@ -274,7 +332,33 @@ Where:
 
 - `session_store_driver`: path to a directory that will hold session
   information.
-- `ownership_voucher_store_driver`: path to a directory that will hold OVs.
+- `ownership_voucher_store_driver`: this selects the ownership voucher storage
+  method. Select between `Directory`, `Sqlite` or `Postgres`.
+    - `Directory`: expects a `path` to the directory that will hold the OVs.
+      For example:
+      ```
+      ownership_voucher_store_driver:
+        Directory:
+          path: /home/fedora/ownership_vouchers
+      ```
+    - `Sqlite`: will use a Sqlite database to store the ownership vouchers.
+      When using this option you must set `Manufacturer` as the DB type as
+      shown below:
+      ```
+      ownership_voucher_store_driver:
+        Sqlite:
+          Manufacturer
```

Comment:
So, for the first part, regarding that you need to specify the database type, I do agree, but it is a bit complicated, but I'll look into it again.

For the postgres-specific part, I went with no configuration parameters since the amount of parameters varies based on how you have configured your server (e.g. password is not always needed, sometimes you need a timeout etc), and in order to make the postgres and sqlite configurations alike, I went with the environment variable option. We also use environment variables to configure some clients, so I'm OK with it.

Reply:
> So, for the first part, regarding that you need to specify the database type, I do agree, but it is a bit complicated, but I'll look into it again.
> 
> For the postgres-specific part, I went with no configuration parameters since the amount of parameters varies based on how you have configured your server (e.g. password is not always needed, sometimes you need a timeout etc), and in order to make the postgres and sqlite configurations alike, I went with the environment variable option. We also use environment variables to configure some clients, so I'm OK with it.

Another possibility, which might be simpler, would be something like:
```yaml
ownership_voucher_store_driver:
  Database:
    URL: "file:///path/to/the/sqlite/db?option1=value1&option2=value2"
```
or
```yaml
ownership_voucher_store_driver:
  Database:
    URL: "postgresql://username:password@host:5432/database_name?option1=value1&option2=value2"
```